dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1059,11 +1068,8 @@ export const Explorer = ({
     isLiveTailOnRef.current,
     patternsData,
     viewLogPatterns,
-    isPatternConfigPopoverOpen,
-    patternRegexInput,
```

Comment:
are these changes required?  this looks like patterns code.

Reply:
resolved some merge conflicts n local, might’ve been deleted during that. Thanks for catching it!